aries_cloudagent/admin/server.py

```diff
@@ -338,6 +362,7 @@ async def start(self) -> None:
     async def stop(self) -> None:
         """Stop the webserver."""
         self.app._state["ready"] = False  # in case call does not come through OpenAPI
+        self.app._state["alive"] = False
```

Comment:
I don't think we should unset `alive` on shutdown.  At this point aca-py is trying to shutdown gracefully and process any remaining work before voluntarily shutting down.  Setting `alive` to false provides k8s with a false indication that could cause it to kill the pod before it gracefully shuts down on it's own.

Reply:
Good catch!